File: Code/4-1/2.py
def add(num1, num2):
    return num1+num2


def sub(num1, num2):
    return num1-num2


def mul(num1, num2):
    return num1*num2


def div(num1, num2):
    return num1/num2


def mod(num1, num2):
    return num1 % num2


def printMsg():
 print("completed")


A = int(input())
B = int(input())

print("sum:", add(A, B))
print("difference:", sub(A, B))
print("product:", mul(A, B))
print("division:", div(A, B))
print("remainder:", mod(A, B))
printMsg()

File: Code/7-1/2.c
#include <stdio.h>

int main()
{
    double num1 = 1.23456789;
    double num2 = 0.0000123456789;

    printf("1) %.10f %.10f\n", num1,num2);
    printf("2) %e %e\n", num1, num2);
    printf("3) %E %E\n", num1, num2);
    printf("4) %g %g\n", num1, num2);
    printf("5) %G %G", num1, num2);

    return 0;
}